### ISSUE TITLE
issue of paper 1

### ISSUE THREAD

**Issue by Jie-jiao05** [DianaShen1224/Donaldson-Paper#2]
Peer Review for Diana Shen

Positive Point
This is a very complete and sufficient paper. In this paper, the comprehensive situation of sheleter in GTA region is fully introduced, and intuitive and clear images are drawn to show each situation. Different data are used to fit in different different images. All references, llm, readme all things are presented, and the sentence are well-organized no obvious mistake. 

Improvement
not too much, it is really a very well-structured paper. But consider something data / analysis ur paper do not conclude and hope could be write, put it at weakness part. also maybe double check with the grammar.

In summary part, maybe u can consider to give a short conclusion of what u find.

Overall very good 60/65

**Comment by DianaShen1224** [comment]
Thank you so much for your review! I have updated a summary /conclusion of the paper in the discussion part, adding the future investigation aspect in the weakness part. Improving the abstract to make it more clear and short. Adding a literature review for the measurement for in-depth. And add comments in the test scripts to guide the readers. Also, improving the formatting and grammar issues of the paper.
